hyperplonk/src/snark.rs

```diff
@@ -337,6 +337,12 @@ where
         // =======================================================================
         // 5. deferred batch opening
         // =======================================================================
+        for eval_point in pcs_acc.points.iter() {
+            transcript.append_serializable_element(b"eval_point", eval_point)?;
+        }
+        for eval in pcs_acc.evals.iter() {
```

Comment:
Maybe it's better to put this line into pcs.multi_open(.) instead, because we always need to FS the evaluation value in a batching proof.

Reply:
wait, just to confirm, you want to only put "append evals" to `multi_open()` but not "append eval_points"? I find it confusing. it makes sense to put all my code addition to PCS batch open (and verify).

Reply:
There is no need to Fiat-Shamir the evaluation points when they're fixed. But yes, we can also choose to Fiat-Shamir them by default.

Reply:
done in [e58574f](https://github.com/EspressoSystems/hyperplonk/pull/174/commits/e58574fda5b8f45b3bbf2c6745b09214c04b26f3)